src/core/lib/channel/http_client_filter.c

```diff
@@ -134,17 +150,57 @@ static void hc_mutate_op(grpc_call_element *elem,
   /* grab pointers to our data from the call element */
   call_data *calld = elem->call_data;
   channel_data *channeld = elem->channel_data;
+
+  /* Decide which HTTP VERB to use */
+  grpc_mdelem *method = GRPC_MDELEM_METHOD_POST;
+  if (op->send_initial_metadata != NULL && (op->send_initial_metadata_flags &
```

Comment:
Comment that we need both send initial metadata and the first send in the same batch (for now) to enable GET